mne_bids_pipeline/scripts/preprocessing/_04b_run_ssp.py

```diff
@@ -156,6 +156,7 @@ def get_config(
         n_proj_eog=config.n_proj_eog,
         n_proj_ecg=config.n_proj_ecg,
         ssp_meg=config.ssp_meg,
+        ch_types=config.ch_types,
```

Comment:
And here you can see a real potential bug due to bad encapsulation, where rejection parameters could depend on ch_types because it was silently used inside config.get_ssp_reject. This PR forces us to actually have this variable in the SimpleNamespace to use it, rather than allowing it to silently sneak in via a config function call.

I can explain Friday if it's not clear